README.md

```diff
@@ -1,159 +1,10 @@
-# sigstore-js &middot; [![npm version](https://img.shields.io/npm/v/sigstore.svg?style=flat)](https://www.npmjs.com/package/sigstore) [![CI Status](https://github.com/sigstore/sigstore-js/workflows/CI/badge.svg)](https://github.com/sigstore/sigstore-js/actions/workflows/ci.yml) [![Smoke Test Status](https://github.com/sigstore/sigstore-js/workflows/smoke-test/badge.svg)](https://github.com/sigstore/sigstore-js/actions/workflows/smoke-test.yml)
+# sigstore-js &middot; [![CI Status](https://github.com/sigstore/sigstore-js/workflows/CI/badge.svg)](https://github.com/sigstore/sigstore-js/actions/workflows/ci.yml) [![Smoke Test Status](https://github.com/sigstore/sigstore-js/workflows/smoke-test/badge.svg)](https://github.com/sigstore/sigstore-js/actions/workflows/smoke-test.yml)
 
-A JavaScript library for generating and verifying Sigstore signatures. One of
-the intended uses is to sign and verify npm packages but it can be used to sign
-and verify any file.
+JavaScript libraries for interacting with [Sigstore][6] services.
 
-## Features
+## Packages
 
-* Support for signing using an OpenID Connect identity
```

Comment:
hm do we want to make these changes to the overall readme? Feels like it's more valuable to focus on the main lib here and maybe link out to the other sub-packages?

Reply:
although can also see how we want a readme in the client folder: packages/client/README.md

Reply:
I didn't want to have to maintain duplicate content between two readmes . . . and the one in the client folder is the one that will appear on the [npm page](https://www.npmjs.com/package/sigstore) so it seemed appropriate to have the library usage information there.